modules/getting-started/ref-audit-log-scaffolder-events.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+// assembly-audit-log.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ref-audit-log-scaffolder-events.adoc_{context}"]
+= Scaffolder events
+
+{product-short} audit logs can include the following scaffolder events:
+
```

Comment:
More for clarification: Are these scaffolder events in a particular order? Or does order not matter?

Reply:
No particular order.